ignite/cmd/network_coordinator_show.go

```diff
@@ -19,15 +20,15 @@ func NewNetworkCoordinatorShow() *cobra.Command {
 }
 
 func networkCoordinatorShowHandler(cmd *cobra.Command, args []string) error {
-	session := cliui.New()
+	session := cliui.New(cliui.StartSpinner())
 	defer session.End()
 
 	nb, err := newNetworkBuilder(cmd, CollectEvents(session.EventBus()))
 	if err != nil {
 		return err
 	}
 
-	n, err := nb.Network()
+	n, err := nb.Network(network.CollectEvents(session.EventBus()))
```

Comment:
We need to pass the `collectEvents` option to both `NetworkBuilder` and `network` ? I would say that passing to `NetworkBuilder` is enough, and it transfers the option to the created `Network`, wdyt?